test/integration/bin/cli.js

```diff
@@ -0,0 +1,53 @@
+const { execSync } = require('node:child_process');
+const uuid = require('uuid').v4;
+
+describe('cli', () => {
+  it('should return status code 1 if no command is issued', () => {
+    let thrown = false; // pattern from test/unit/util/crypto.js
+    try {
+      // eslint-disable-next-line no-use-before-define
+      cli('', { stdio: ['pipe', 'pipe', 'ignore'] });
+    } catch (err) {
+      err.status.should.equal(1);
+      thrown = true;
+    }
+    thrown.should.equal(true);
+  });
+
+  it('should return status code 1 if non-existent command is issued', () => {
+    let thrown = false; // pattern from test/unit/util/crypto.js
+    try {
+      // eslint-disable-next-line no-use-before-define
+      cli('user-congratulate', { stdio: ['pipe', 'pipe', 'ignore'] });
+    } catch (err) {
+      err.status.should.equal(1);
+      thrown = true;
+    }
+    thrown.should.equal(true);
+  });
+
+  describe('command: user-create', () => {
+    it('should return status code 0 and user details on success', () => {
```

Comment:
It feels unclear to me what should be tested in this file vs. test/integration/task/account.js. It makes sense that this file is testing status codes. For testing output, my instinct is that that should be done in test/integration/task/account.js along with the other tests related to the behavior of each task.

Reply:
This is an integration test for `lib/bin/cli.js`.  I'd be surprised to find it in `test/integration/task/account.js`.  Perhaps `cli.js` is named misleading though, as it only deals with accounts(?)This is an integration test for `lib/bin/cli.js`.  I'd be surprised to find it in `test/integration/task/account.js`.  Perhaps `cli.js` is named misleading though, as it only deals with accounts(?)

Reply:
I think that's right that cli.js only deals with accounts. It may have been named the way it was with the thought that it would eventually do more things. In the [user docs](https://docs.getodk.org/central-command-line/), we mention an `odk-cmd`, which just calls cli.js I think. As far as I know, the only other script that users run directly is restore.js for restoring a backup ([user docs](https://docs.getodk.org/central-backup/)).